typesense-java/src/main/java/org/typesense/model/Client.java

```diff
@@ -0,0 +1,37 @@
+package org.typesense.model;
+
+import java.util.HashMap;
+
+public class Client {
+    private Configuration configuration;
+
+    private Api api;
+
+    private Collections collections;
+
+    private HashMap<String, Collection> individualCollections;
+
+    Client(Configuration configuration){
+        this.configuration = configuration;
+        this.api = new Api(configuration);
+        collections = new Collections(api);
+        this.individualCollections = new HashMap<>();
+    }
+
+    public Collection collections(String name){
+        Collection ret_val;
+
+        if(!this.individualCollections.containsKey(name)){
```

Comment:
We should probably commit a common code style/formatter for IDEA/Eclipse/NetBeans where possible, tends to cut down on noise in the git history.